### DISCUSSION TITLE
I'm trying to put in custom fields in the user profile for pnp modern search and need multi-select with quotation marks

### DISCUSSION THREAD

**Discussion by KProuty** [microsoft-search/pnp-modern-search#3798] [Q&A]
I've read a bunch of stuff and I'm close but I am NOT a programmer. I am a power user.

 
I'm taking a multi-select lookup column of Programs from a sharepoint list that is tied to a user and want to put that in a custom field on the profile. Programs is multi-select similar to Past Projects in the Sharepoint User Profile Service. I've got everything working on the filters, mapping, etc. but the flow to populate the field correctly from the list is the last piece.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30842503/91d4cee2-7a10-463b-82dd-1516e164be60)

 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30842503/530e2ad7-334c-44fe-ada9-5bf19cc200f2)

 

And I have it so it gets rid of the commas and puts in semicolons:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30842503/745a308d-aef8-47ff-bf44-113398d03734)

But for the life of me I can't get the values between the delimiters to have double quotes so it meets this requirement to work (got this from another post on here, [Right here](https://powerusers.microsoft.com/t5/Building-Flows/Need-help-building-a-flow-to-update-a-sharepoint-user-profile/m-p/2436131) !)

 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30842503/333e2d5f-91be-4839-95f1-4ac8e3dddc4d)

 

This is my current output:

[
"value":ETools; Special Service; Conv for Caring
]

and I need it to be:

 
"GLSSPrograms";["ETools"; "Special Service"; "Conv for Caring"]

 
Any help truly appreciated. I'm sure a real programmer could do this no problem, but I am NOT that. I am a copyist and everyone assumes people have the basic programming skills. I just haven't found the secret video that walks this through step by step.

Thank you!

**Comment by kasperbolarsen** [comment]
I would recommend asking this question in PowerApps related forum. This forum is all about Search and PnP Modern Search in particular, and has nothing to do with PowerApps and User Profiles.
Best of luck
